packages/admin/blocks-admin/src/blocks/VimeoVideoBlock.tsx

```diff
@@ -0,0 +1,112 @@
+import { Field, FieldContainer, FinalFormInput, FinalFormRadio, FinalFormSwitch } from "@comet/admin";
+import { Box, FormControlLabel } from "@mui/material";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { VimeoVideoBlockData, VimeoVideoBlockInput } from "../blocks.generated";
+import { BlocksFinalForm } from "../form/BlocksFinalForm";
+import { SelectPreviewComponent } from "../iframebridge/SelectPreviewComponent";
+import { useAdminComponentPaper } from "./common/AdminComponentPaper";
+import { createBlockSkeleton } from "./helpers/createBlockSkeleton";
+import { BlockCategory, BlockInterface } from "./types";
+
+const isValidVimeoIdentifier = (value: string) => {
+    const urlRegEx = /(https?:\/\/)?(www\.)?(player\.)?vimeo\.com\/?(showcase\/)*([0-9)([a-z]*\/)*([0-9]{6,11})[?]?.*/;
+    const idRegEx = /([0-9]{6,11})/;
+
+    const urlMatch = value.match(urlRegEx);
+    const idMatch = value.match(idRegEx);
+
+    return !!urlMatch || !!idMatch;
+};
+
+const validateIdentifier = (value?: string) => {
+    return value && isValidVimeoIdentifier(value) ? undefined : (
+        <FormattedMessage id="comet.blocks.vimeoVideo.validation" defaultMessage="Should be a valid Vimeo URL or identifier" />
+    );
+};
```

Comment:
Please change the validation to now show an error for empty input fields. (as in #2121)

Reply:
https://github.com/vivid-planet/comet/pull/2068/commits/434fffca7cce3a8ea358d2c3b53c175efa0a5335